### DISCUSSION TITLE
Native window host

### DISCUSSION THREAD

**Discussion by ArkasisProjects** [tauri-apps/tauri#7545] [Q&A] [answered]
I understand that this question might seem stupod, but I'm curious to know if there's a practical method to incorporate a native window's rendering within a Tauri application. To elaborate, let's say I have a window with an OpenGL context. Is there a feasible approach to display this window's rendering within the Tauri window? My goal is to create an "editor" using tauri for an app that is made with GLFW.

**Comment by f1forhelp** [comment]
As a Flutter developer, I believe this feature is essential; otherwise, Tauri risks being similar to IONIC. Platform views are crucial for building truly cross-platform apps. It's important to have the ability to host native platform views in Tauri, such as an Android-specific video player.

@lucasfernog @amrbashir @FabianLars

**Reply by lucasfernog** [reply, marked as answer]
this is not possible yet, and will require a lot of effort to be supported.
